libobs/obs-output.h

```diff
@@ -28,8 +28,13 @@ extern "C" {
 #define OBS_OUTPUT_SERVICE (1 << 3)
 #define OBS_OUTPUT_MULTI_TRACK (1 << 4)
```

Comment:
Wonder if we can somehow apply a DEPRECATED flag to this to throw a warning for anything using the old name?

Reply:
Yeah, I thought about this but couldn't come up with anything that fit that usage pattern in a macro